shop3/docs/manual.texinfo

```diff
@@ -586,14 +587,15 @@ functionality is inspired, in part, by Anytime Algorithms [Dean and
 Boddy, 1998].
 
 @item @anchor{#explanation}
-If @emph{explanation} is non-nil, @sysname {} adds extra information at the
-end of each operator explaining how the preconditions for that operator
-were satisfied. Currently supports only logical atoms, and, and or; it
-doesn't work with forall, not, eval, etc. If this feature is used with
-the external-access-hook feature (see Section 5.4), any attribution
-information provided by the external-access-hook routine is included in
-the relevant explanation. The default value of @emph{explanation} is
-@code{nil}. 
+If @emph{explanation} is non-nil, @sysname {} adds extra information
+at the end of each operator explaining how the preconditions for that
+operator were satisfied. Currently supports only logical atoms,
+@code{and}, and @code{or}; it doesn't work with @code{forall},
+@code{not}, @code{eval}, etc. If this feature is used with the
+external-access-hook feature (see @ref{Hook Routines}), any attribution
```

Comment:
Should change to mark external-access-hook as code. Also add an index entry for it here.